cmd/kube-apiserver/app/testing/testserver.go

```diff
@@ -86,19 +89,18 @@ type Logger interface {
 	Logf(format string, args ...interface{})
 }
 
+type ProxyCA struct {
+	ProxySigningCert *x509.Certificate
+	ProxySigningKey  *rsa.PrivateKey
+}
+
 // NewDefaultTestServerOptions Default options for TestServer instances
 func NewDefaultTestServerOptions() *TestServerInstanceOptions {
 	return &TestServerInstanceOptions{
 		EnableCertAuth: true,
 	}
 }
 
-// StartTestServer starts a etcd server and kube-apiserver. A rest client config and a tear-down func,
-// and location of the tmpdir are returned.
-//
-// Note: we return a tear-down func instead of a stop channel because the later will leak temporary
-// files that because Golang testing's call to os.Exit will not give a stop channel go routine
-// enough time to remove temporary files.
 func StartTestServer(t Logger, instanceOptions *TestServerInstanceOptions, customFlags []string, storageConfig *storagebackend.Config) (result TestServer, err error) {
```

Comment:
Why remove this godoc?

Reply:
Sorry, this was not intentional. Reverted.